### ISSUE TITLE
Loss function question

### ISSUE THREAD

**Issue by issakh** [HyeongminLEE/AdaCoF-pytorch#5]
Hi, the default loss function is set to 1*Charb+0.01*g_Spatial+0.005*g_Occlusion'. Charb is the Charbonnier loss, but I can't seem to figure out what g_spatial and g_occlusion are based on the code?
For the adversarial loss, of the options stated (GAN, WGAN, WGAN_GP, T_WGAN_GP, FI_GAN) which was the one used to get the best results in the paper?
Thanks for your help

**Comment by HyeongminLEE** [comment]
1. g_spatial and g_occlusion is the smoothness constraint terms for flow maps and occlusion maps. However, since these terms did not significantly change the result, we removed these terms from the paper. You can freely remove these terms.

2. FI_GAN is the proposed version of the paper.

Thanks.
